packages/perseus/src/components/graph.tsx

```diff
@@ -38,7 +39,46 @@ function numSteps(range: any, step: any) {
     return Math.floor((range[1] - range[0]) / step);
 }
 
-class Graph extends React.Component<any> {
+type Props = {
+    box: [number, number];
```

Comment:
Is the reason we're not making these `Readonly` because we want them to be fixed length? Or do we want this to be mutable?

Because if we want them to be both `Readonly` and fixed length, could we use:

```suggestion
    box: Readonly<[number, number]>;
```